### DISCUSSION TITLE
Project update at TAG Storage meeting

### DISCUSSION THREAD

**Discussion by xing-yang** [k8up-io/k8up#823] [General]
Hi k8up maintainers,

I'm a co-chair of [CNCF TAG Storage](https://github.com/cncf/tag-storage). I think it would be great for you to give a project overview of k8up at one of our TAG meetings.

K8up was donated to CNCF as a sandbox project in November 2021 as a Cloud Native Storage project. We have not seen a presentation about it at the TAG. It would be great to learn things like the roadmap, tech, and community of the project.

Let me know if this is something you'd be interested in doing. If yes, please feel free to add it to our [agenda](https://docs.google.com/document/d/1DigEag4UUpf53qYBEr50YIdVJJvhXhxHN5ATj-js-IA/edit) or reach out to us in the #tag-storage channel on the CNCF slack.

Thanks!

cc @[chira001](https://github.com/chira001)

**Comment by tobru** [comment]
Thank you very much for getting in touch with us. We'll be happy to present K8up at one of the upcoming TAG meetings. I'll make sure to put us on your agenda :+1:

**Comment by xing-yang** [comment]
Sounds good.  Thank you @tobru!

**Reply by tobru** [reply]
FYI We'll try to find a slot this summer, before we won't be able to join. And we'll be at KubeCon.
